### PR TITLE
Jan 2016

### PR DESCRIPTION
**Pull request by aaronj1335**
@lawnsea

### CODE REVIEW

File: _posts/2016-01-19-alex-sexton.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <img alt="Alex Sexton @slexaxton on Twitter" src="https://avatars1.githubusercontent.com/u/96554?v=3&s=400" />
+  </a>
+  <div>
+  <a href="https://twitter.com/slexaxton"><strong>Alex Sexton</strong></a>
+
+  is no less than the Official Apple-Flavored Hi-Chew Brand Ambassador. If he ever runs for office, this post was fake.
+  </div>
+</div>
+
+Thanks [&#128056; design][frog] for hosting, as always you folks are [great][]. Also if you have a slice of pizza or a drink, be sure to find one of the good folks from [The Iron Yard][]!
```
Comment (lawnsea) [inline _posts/2016-01-19-alex-sexton.md]: Find them and do what to them? :hocho:?

File: _posts/2016-01-19-alex-sexton.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+README: >
+  as soon as we know the date for the next meetup, `cp _drafts/meetup.md
+  _posts/YYYY-MM-DD-meetup.md`, remove this README, and add a `when` property to this
+  section. the front page will update to let folks know that we've got
+  something in the works.
```
Comment (lawnsea) [inline _posts/2016-01-19-alex-sexton.md]: Should we remove the `README`?

**Comment by lawnsea** [comment]
:+1:

**Comment by aaronj1335** [comment]
@lawnsea i'll go ahead and respond to the previous tweet w/ the blog post

**Comment by beaktor** [comment]
:+1:

**Comment by aaronj1335** [comment]
thnx @beaktor !

**Pull request closed without merging**